new-integration-tests/src/test/resources/OperatorIntegrationTests.properties

```diff
@@ -0,0 +1 @@
+WLSIT-0000=WLS Operator Integration Tests
```

Comment:
can we use `WKO-TEST-0000`??

Reply:
Doubt we will ever use it at all, but sure.